sympy/core/power.py

```diff
@@ -1037,9 +1037,15 @@ def _eval_expand_power_exp(self, **hints):
             if isinstance(e, Sum) and e.is_commutative:
                 from sympy.concrete.products import Product
                 return Product(self.func(b, e.function), *e.limits)
-        if e.is_Add and e.is_commutative:
-            return Mul(*[self.func(b, x) for x in e.args])
-        return self.func(b, e)
+        if e.is_Add and (b.is_zero is False or e._all_nonneg_or_nonppos()):
```

Comment:
Perhaps `force` should be checked here to allow overriding this when desired.

Reply:
done

Reply:
I wasn't sure if `posify` should be used on the expression or `force` should be used; but given the case of `x**(y - z)` which would not be resolved by `posify`, I think `force` is the right option.

Reply:
I think that `posify` is usually a bad idea. Here we use `force` to avoid a degenerate case. There are infinitely many possible values that the base could take and the formula is valid for almost all of them. Using `force=True` says "I don't care about that one exceptional case" (when the base is zero). Most of the time that will be a justifiable assumption.

Assuming something to be positive is not the same at all. If we imagine that all real numbers are "equally likely" then assuming that a symbol is positive is going to be wrong 50% of the time. That's very different from overlooking a degenerate case.